src/applications/pact-act/containers/questions/TernaryRadios.jsx

```diff
@@ -122,7 +124,9 @@ const TernaryRadios = ({
               </va-additional-info>
             </div>
           )}
+          {textBeforeList && <p>{textBeforeList}</p>}
```

Comment:
The new Burn Pit question has text before and after the bullet point (unlike any other question).